zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/service/ObjectManagementService.kt

```diff
@@ -98,16 +104,174 @@ class ObjectManagementService(
         )
 
         val objectsListDto = objectsList.results.map {
-            ObjectsListRowDto(it.url.toString(), listOf(
-                ObjectsListRowDto.ObjectsListItemDto("objectUrl", it.url),
-                ObjectsListRowDto.ObjectsListItemDto("recordIndex", it.record.index),
+            ObjectsListRowDto(
+                it.url.toString(), listOf(
+                    ObjectsListRowDto.ObjectsListItemDto("objectUrl", it.url),
+                    ObjectsListRowDto.ObjectsListItemDto("recordIndex", it.record.index),
 
-            ))
+                    )
+            )
         }
 
         return PageImpl(objectsListDto, pageable, objectsList.count.toLong())
     }
 
+    fun getObjectsWithSearchParams(
+        searchWithConfigRequest: SearchWithConfigRequest,
+        id: UUID,
+        pageable: Pageable
+    ): PageImpl<ObjectsListRowDto> {
+        val objectManagement = getById(id) ?: let {
+            logger.info {
+                "The requested Id is not configured as a objectnamagement configuration. " +
+                    "The requested id was: $id"
+            }
+            throw NotFoundException()
+        }
+
+        val searchFieldList = searchFieldV2Service.findAllByOwnerId(id.toString())!!
+
+        val searchDtoList = searchFieldList.flatMap { searchField ->
+            searchWithConfigRequest.otherFilters
+                .filter { otherFilter -> otherFilter.key == searchField.key }
+                .flatMap { otherFilter ->
+                    if (searchField.fieldType != FieldType.RANGE) {
+                        otherFilter.values.flatMap { value ->
```

Comment:
Please throw an error when there are multiple `otherFilter.values`. With error:
`Unsupported operation exception: the objecten API doesn't support 'OR' searches`

The values in the 'otherFilter.values' are supposed to be done with an `OR` search. The Multi-select-dropdown makes use of that. But the objectenAPI can't do that